presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -13,6 +13,8 @@
  */
 
 #include "presto_cpp/main/common/Configs.h"
+#include <string>
```

Comment:
Do we need this? Can we separate the system include from project include? Putting this line at top and en empty line below to separate would be nice.